docs/dev-setup.md

```diff
@@ -111,5 +111,34 @@ spec:
 The output can be passed directly to `oc apply` like this:
 
 ```
-$ go run cmd/make-worker/main.go openshift_worker_1 | oc apply -f -
+$ go run cmd/make-virt-worker/main.go openshift_worker_1 | oc apply -f -
+```
+
+## Using Bare Metal Hosts
+
+The `make-bm-worker` tool may be a more convenient way of creating
+YAML definitions for workers than editing the files directly.
+
+```
+$ go run cmd/make-bm-worker/main.go -address 1.2.3.4 -password password -user admin worker-99
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: worker-99-bmc-secret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: cGFzc3dvcmQ=
+
+---
+apiVersion: metalkube.org/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: worker-99
+spec:
+  online: true
+  bmc:
+    address: 1.2.3.4
```

Comment:
I guess it assumes IPMI without a prefix?

Reply:
Yes, see https://github.com/metalkube/baremetal-operator/blob/master/pkg/bmc/access.go#L49